mne/epochs.py

```diff
@@ -85,7 +85,7 @@ def _save_split(epochs, fname, part_idx, n_parts):
 
     # write events out after getting data to ensure bad events are dropped
     data = epochs.get_data()
-    assert data.dtype == 'float64'
+    assert data.dtype in ['float64', 'complex64', 'complex128']
```

Comment:
This is weird. Is this how we do it for raw data? I would have expected to store as `float32` and `complex64` and convert to `float64` and `complex128` in memory.

Reply:
Where is this conversion happening? After hilbert-transform the datatype is `complex128`.